front/app/containers/InitiativesShow/ReactionControl/Status/ProposedReacted.tsx

```diff
@@ -99,18 +61,31 @@ const ProposedReacted = ({
   initiative,
   initiativeSettings: { reacting_threshold },
   onCancelReaction,
+  onReaction,
+  userReacted,
 }: StatusComponentProps) => {
   const reactionCount = initiative.attributes.likes_count;
   const reactionLimit = reacting_threshold;
   const daysLeft = getPeriodRemainingUntil(initiative.attributes.expires_at);
 
   return (
     <Container>
-      <StyledIcon ariaHidden name="check-circle" />
-      <ReactedTitle>
-        <FormattedMessage {...messages.votedTitle} />
-      </ReactedTitle>
-      <ReactedText>
+      <Box ml="auto">
+        <CountDown targetTime={initiative.attributes.expires_at} />
```

Comment:
Just streamlining UIs: whenever voting is going on, we show the countdown (still checking with prod later but can merge for now).